installer/Plugins.wxs

```diff
@@ -199,7 +184,28 @@
       <ComponentRef Id="ut3.dll" />
       <ComponentRef Id="wolfet.dll" />
       <ComponentRef Id="wow.dll" />
-      <ComponentRef Id="wow_x64.dll" />
     </ComponentGroup>
+    <?if $(sys.BUILDARCH) = "x64" ?>
```

Comment:
Is it valid to redefine the PluginFolder directoryref, and the Plugins componentgroup? This seems wrong.

I would simply group the x64 plugins together inside `<?if $(sys.BUILDARCH) = "x64" ?>...<?endif >` in each of them (one on the DirectoryRef, one in the ComponentGroup)...

Does this build?